src/core/lib/security/security_connector/load_system_roots_supported.cc

```diff
@@ -62,13 +62,13 @@ const char* kCertFiles[] = {"/etc/ssl/cert.pem"};
 const char* kCertDirectories[] = {""};
 #endif                      // GPR_APPLE
 
-grpc_slice GetSystemRootCerts() {
+Slice GetSystemRootCerts() {
   size_t num_cert_files_ = GPR_ARRAY_SIZE(kCertFiles);
```

Comment:
Pre-existing nit: This should not have an underscore suffix, since it's not a class data member.